plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
AMD releases a driver every 2 weeks. They have several branches. Upgrading the driver is mandatory if you want to enjoy new game. I don't see the issue to ask to upgrade their driver if rendering is broken. It would help to fix bug in others applications too.

I used the same politic for Intel/Nvidia/Free driver bugs. So far nobody complain. (well it helps that they release driver faster). So I don't see the point to bother.